core/deployment/src/main/java/io/quarkus/deployment/steps/SubstrateConfigBuildStep.java

```diff
@@ -104,6 +105,27 @@ void build(SslContextConfigurationTemplate sslContextConfigurationTemplate,
                 sslTrustStoreSystemProperty.produce(
                         new SslTrustStoreSystemPropertyBuildItem(
                                 graalVmLibDirectory.resolve(Paths.get("security", "cacerts")).toString()));
+
+                boolean enableJni = false;
+                for (JniBuildItem jniBuildItem : jniBuildItems) {
+                    enableJni = true;
+                    if (jniBuildItem.getLibraryPaths() != null && !jniBuildItem.getLibraryPaths().isEmpty()) {
+                        for (String path : jniBuildItem.getLibraryPaths()) {
+                            Path libraryPath = Paths.get(path);
+                            if (!libraryPath.isAbsolute()) {
+                                Path absoluteLibPath = libraryPath.toAbsolutePath();
```

Comment:
Thinking more about it, I have a question about this: do we really need to make the path absolute?

Because it will make it quite difficult to build on one machine and push to another?